cmd/bots/internal/tg_bot/bot.go

```diff
@@ -0,0 +1,47 @@
+package tg_bot
+
+import (
+	"gopkg.in/telebot.v3"
+	"log"
+	"nodemon/pkg/entities"
+	"nodemon/pkg/storing/chats"
+)
+
+type TelegramBotEnvironment struct {
+	ChatStorage *chats.Storage
+	Bot         *telebot.Bot
+	ShutUp      bool
```

Comment:
I think we should be a bit more polite. Rename this variable to a corresponding neutral word.